File: app/src/main/java/org/miaowo/miaowo/bean/data/web/TopicList.java
package org.miaowo.miaowo.bean.data.web;

import java.util.List;

public class TopicList {

    /**
     * tags : [{"value":"猜猜我是谁","score":2,"color":"","bgColor":""},{"value":"好奇喵app","score":1,"color":"","bgColor":""},{"value":"这是测试","score":0,"color":"","bgColor":""}]
     * nextStart : 100
     * breadcrumbs : [{"text":"[[global:home]]","url":"/"},{"text":"[[tags:tags]]"}]
     * title : [[pages:tags]]
     * loggedIn : true
     * relative_path :
     * template : {"name":"tags","tags":true}
     * url : /tags
     * bodyClass : page-tags
     */

    private int nextStart;
    private String title;
    private boolean loggedIn;
    private String relative_path;
    private String url;
    private String bodyClass;
    private List<Topic> tags;

    public int getNextStart() {
        return nextStart;
    }

    public void setNextStart(int nextStart) {
        this.nextStart = nextStart;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public boolean isLoggedIn() {
        return loggedIn;
    }

    public void setLoggedIn(boolean loggedIn) {
        this.loggedIn = loggedIn;
    }

    public String getRelative_path() {
        return relative_path;
    }

    public void setRelative_path(String relative_path) {
        this.relative_path = relative_path;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public String getBodyClass() {
        return bodyClass;
    }

    public void setBodyClass(String bodyClass) {
        this.bodyClass = bodyClass;
    }

    public List<Topic> getTags() {
        return tags;
    }

    public void setTags(List<Topic> tags) {
        this.tags = tags;
    }
}

File: app/src/main/java/org/miaowo/miaowo/bean/data/TopicList.kt
package org.miaowo.miaowo.bean.data

/**
 * 话题列表
 *
 * 忽略的属性：
 * template : {"name":"tags","tags":true}
 * breadcrumbs : [{"text":"[[global:home]]","url":"/"},{"text":"[[tags:tags]]"}]
 */
data class TopicList (
        var nextStart: Int = 0,  // nextStart : 100
        var title: String? = null,  // title : [[pages:tags]]
        var isLoggedIn: Boolean = false,  // loggedIn : true
        var relative_path: String? = null,  // relative_path :
        var url: String? = null,  // url : /tags
        var bodyClass: String? = null,  // bodyClass : page-tags
        var tags: List<Topic>? = null  // tags : [{"value":"猜猜我是谁","score":2,...}]
)